modules/mobianRtdProvider.js

```diff
@@ -0,0 +1,53 @@
+import { submodule } from '../src/hook.js';
+import { ajaxBuilder } from '../src/ajax.js';
+
+export const MOBIAN_URL = 'http://impact-analytics-staging.themobian.com';
+
+export const mobianBrandSafetySubmodule = {
+  name: 'mobianBrandSafety',
+  gvlid: null,
+  init: init,
+  getTargetingData: getTargetingData
+};
+
+function init() {
+  return true;
+}
+function getTargetingData() {
```

Comment:
This doesn't appear to make these data available in the bid request object

I would suggest site.ext.data.mobian

Reply:
Thanks for the prompt feedback! After doing some reading up, it seemed like the best way to access and insert data into `site.ext.data` was via `getBidRequestData`, which is the current approach.